router/src/validation.rs

```diff
@@ -217,7 +217,7 @@ impl Validation {
         }
 
         let temperature = temperature.unwrap_or(1.0);
-        if temperature <= 0.0 {
+        if temperature < 0.0 {
```

Comment:
No temperature = 0 still needs to yell on the regular URLs. It yields incorrect results if you really use temperature==0 as an exponent.

Reply:
this was removed in the latest change since `temperature` should never be 0 at this point